Formula/sifu.rb

```diff
@@ -0,0 +1,22 @@
+class Sifu < Formula
+  desc "Rapid software development tool made for creating cloud solutions."
```

Comment:
Basically we are generating custom code from user specifications in witch they describe basic functionalities of their servers and clients. You can see more details in the link
https://codesifu.com/gettingstarted.html